reference/supported-languages.md

```diff
@@ -0,0 +1,36 @@
+# Languages/Formats OpenRewrite supports
+
+OpenRewrite is continuously adding support for new languages and formats. As of June 2023, OpenRewrite supports:
+
+## Programming languages
+
+* Java
+* Groovy/Gradle 
+* Kotlin
```

Comment:
```suggestion
* Kotlin
* Groovy
```
Gradle isn't as much a language as it is a build tool, much like Maven. It might be slightly confusing to have Gradle mentioned here.
And Kotlin is gaining traction, so figured list that above Groovy to attract the youngsters. ;)

Reply:
haha fair enough :D